app/assets/stylesheets/topics.scss

```diff
@@ -1,3 +1,13 @@
 // Place all the styles related to the topics controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+@media print {
+  #pipeline-wrapper {
+    display: none;
+  }
+
+  #left-col-user-info {
+    display: none;
+  }
+}
```

Comment:
@hin101 `scss-lint` is complaining about there not being a new line at the end of this file!